dod-advana/gamechanger-web
# UOT-106416 EDA Get Line Items (PG) to Export

## Description

- Karolyn: add/connect to EDA database and create query to retrieve line items
- update EDA query to use models and sequelize
- add line items to the EDA export data
- 
## Related Issue/Ticket

https://confluence.di2e.net/display/UOT/UOT-106416+EDA+Get+Line+Items+%28PG%29+to+Export


## Testing instructions

- make sure you have the EDA .env configs (can ask Aaron)
- navigate to /#/contractsearch (coupled only)
- make any search
- use the export function (JSON and CSV)
- check for line item data (CLINS)
- for CSV: should see columns H - N somewhat populated with data
- for JSON: look for data under "line_items" (I drag the json into VS Code and use Prettify JSON to view)
